concordium-contracts-common/src/schema_json.rs

```diff
@@ -1087,6 +1223,140 @@ mod tests {
 
     use super::*;
 
+    /// Tests schema template display in JSON of an Enum
+    #[test]
+    fn test_schema_template_enum() {
+        let schema = Type::Enum(Vec::from([
+            (String::from("Accounts"), Fields::Unnamed(Vec::from([Type::AccountAddress]))),
+            (String::from("Accounts"), Fields::Unnamed(Vec::from([Type::AccountAddress]))),
+        ]));
+        assert_eq!(
+            json!({"Enum": [{"Accounts": ["<AccountAddress>"]},{"Accounts": ["<AccountAddress>"]}]}),
+            schema.to_json_template()
```

Comment:
This seems odd. Both variants have the same name. Perhaps you meant to display an `Address` with `AddressAccount` / `AddressContract` variants?